### PR TITLE
(#408) Remove redux from PrintModalContent

### LINKED ISSUE #408: Update PrintModalContent.jsx to remove Redux
> **ESTIMATE** 2

The print modal uses redux to get the search criteria in order to pass over the original query to the print API. This should be a quick fix to pass in the searchCriteria and remove Redux.

### PR DESCRIPTION
**Pull request by kate-mashkina**
- removed unused props from
PrintModalContent
- removed redux from PrintModalContent.jsx

Closes #408

### CODE REVIEW

File: src/views/ResultsPage/PrintModalContent.jsx [hunk 1]
```diff
@@ -124,5 +121,6 @@ PrintModalContent.propTypes = {
 	selectedList: PropTypes.array,
 	handleClose: PropTypes.func,
 	closeModal: PropTypes.func,
+	searchCriteriaObject: PropTypes.object,
```
Comment (seyilonge-nci) [inline src/views/ResultsPage/PrintModalContent.jsx:124]: Can you update to `.isRequired` for this and `selectedList` above as well

File: src/views/ResultsPage/PrintModalContent.jsx [hunk 2]
```diff
@@ -124,5 +121,6 @@ PrintModalContent.propTypes = {
 	selectedList: PropTypes.array,
 	handleClose: PropTypes.func,
```
Comment (seyilonge-nci) [inline src/views/ResultsPage/PrintModalContent.jsx:122]: We have `handleClose` and `closeModal` which I believe should be doing the same thing, but they aren't even used in this component. Know it's not a change you made, just raising the question.

Comment (kate-mashkina) [reply src/views/ResultsPage/PrintModalContent.jsx:122]: I agree with this one - 	handleClose  is being passed into component from Res page but it is not used in component itself.. I did not touch it as part of this ticket, unless this is a bad code and we want to fix it up

Comment (arcepaul) [reply src/views/ResultsPage/PrintModalContent.jsx:122]: You can go ahead and remove from proptypes as it isn't being handled.  This is probably artifact from a previous implementation.

Comment (kate-mashkina) [reply src/views/ResultsPage/PrintModalContent.jsx]: It is gone @arcepaul

File: src/views/ResultsPage/PrintModalContent.jsx [hunk 3]
```diff
@@ -1,20 +1,17 @@
...
 import PropTypes from 'prop-types';
-import { useSelector } from 'react-redux';
 import { usePrintApi } from '../../hooks';
 import { buildQueryString } from '../../utilities';
 import { useAppSettings } from '../../store/store.js';
 
 const queryString = require('query-string');
 
-const PrintModalContent = ({ selectedList = [] }) => {
+const PrintModalContent = ({ selectedList = [], searchCriteriaObject }) => {
```
Comment (arcepaul) [inline src/views/ResultsPage/PrintModalContent.jsx:9]: NIT: just realized, selected list is required, no need for defaulting to empty string.

Comment (kate-mashkina) [reply src/views/ResultsPage/PrintModalContent.jsx]: done!

**Review by arcepaul — APPROVED** [review]

**Pull request merged**